CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/#semantic-versioning-200).
 
+## [2.3.6] - ?
+
+### :crab: Changed
+- Changed log level of `Failover.startWriterFailover` and `Failover.establishedConnection` to info ([PR #922](https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/922))
```

Comment:
```suggestion
- Log level of `Failover.startWriterFailover` and `Failover.establishedConnection` from `fine` to `info` ([Issue #890](https://github.com/awslabs/aws-advanced-jdbc-wrapper/issues/890)).
```
Using the word Change in the beginning is a bit redundant, so i removed that. 
Also, if the change is tied to an issue, it is preferable to link the issue, instead of the PR. 

It might also be nice to briefly add a reason for the change.

Reply:
Updated